programs/test/selftest.c

```diff
@@ -246,6 +246,9 @@ const selftest_t selftests[] =
 #if defined(MBEDTLS_SHA256_C)
     {"sha256", mbedtls_sha256_self_test},
 #endif
+#if defined(MBEDTLS_SHA384_C)
+    {"sha512", mbedtls_sha384_self_test},
```

Comment:
Shouldn't the string be `"sha384"`?

Reply:
ops, you're right. Besides I also found a similar issue few lines above for the sha224. I'll fix both ;)